backend/src/main/resources/application.properties

```diff
@@ -1,8 +1,22 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/SkyNest?createDatabaseIfNotExist=true
+
+#   MySQL
+spring.datasource.url=jdbc:mysql://localhost:3305/SkyNest?createDatabaseIfNotExist=true
 spring.datasource.username=user
 spring.datasource.password=123456
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5InnoDBDialect
 spring.flyway.baseline-on-migrate=true
 #spring.jpa.hibernate.ddl-auto=none
 #spring.jpa.show-sql: true
-#springdoc.swagger-ui.path=/swagger-ui.html
\ No newline at end of file
+
+#   Docs
+#springdoc.swagger-ui.path=/swagger-ui.html
+
+#   Cassandra
+spring.data.cassandra.contact-points=localhost
+spring.data.cassandra.port=9042
```

Comment:
Since you are using values here and in docker file as well, maybe it is good option to consider extract those values in single file. If not with this PR, mark as TODO and create a technical dept task for it in backlog or something